CommunicationLayer/MessagingFramework/CmuMessage.cpp

```diff
@@ -0,0 +1,76 @@
+#include "CmuMessage.h"
+#include "MessageDecodingHelpers.h"
+#include "MessageDefines.h"
+
+namespace
+{
+    const int CELL_NUMBER_INDEX = 1;
+    const int CELL_VOLTAGE_0_INDEX = 2;
+    const int PCB_TEMPERATURE_INDEX = 18;
+    const int CELL_TEMPERATURE_0_INDEX = 20;
+    const int NUMBER_OF_VOLTAGES = 8;
+    const int NUMBER_OF_TEMPERATURES = 15;
+    const int NUMBER_OF_BYTES_IN_UNSIGNED_SHORT = 2;
```

Comment:
Small thing, `UNSIGNED_SHORT_LEN` would be a decent and shorter name for this variable.